package.json

```diff
@@ -58,6 +58,7 @@
     "@fortawesome/free-regular-svg-icons": "^5.11.2",
     "@fortawesome/free-solid-svg-icons": "^5.11.2",
     "@fortawesome/react-fontawesome": "^0.1.7",
+    "audit": "0.0.6",
```

Comment:
Co to za pakiet i dlaczego go dodałaś?

Reply:
w zadaniu jest napisane aby dodac npm audit, tak wiec zrobilam npm install audit

Reply:
W zadaniu napisane jest aby korzystając z `npm audit` przeprowadzić aktualizację pakietów. Nie ma mowy o instalacji pakietu audit. Usuń to proszę. 

Komenda `npm audit` jest wbudowane w komendzie `npm` i wystarczy wpisać w terminalu komendę `npm audit` . Polecam krótkie video https://www.youtube.com/watch?v=rXV6-eXB26s

Reply:
w takim razie usunac?

Reply:
wiesz co chyba usune wszystko i zrobie na nowo